openedx/features/custom_fields/multiselect_with_other/forms/fields.py

```diff
@@ -37,8 +38,10 @@ def valid_value(self, value):
         return len(value) <= self.other_max_length
 
     def validate(self, value):
-        """
-        Validate that the input is a list or tuple.
+        """ Validate that the input is a list or tuple.
+        :param value: list of selected choices
+        :type value: list
+        :raise ValidationError: Validation Error
         """
         if self.required and not value:
             raise ValidationError(self.error_messages['required'], code='required')
```

Comment:
We need to document when/why `ValidationError` is raised

One example here could be 
`:raise ValidationError: Raise validation error when required value not provided`